test/erl_optics_unit_tests.erl

```diff
@@ -0,0 +1,139 @@
+-module(erl_optics_unit_tests).
+-include_lib("eunit/include/eunit.hrl").
+
+start_stop_test() ->
+    start(),
+    check_in_foil(optics),
+    stop(),
+    ?assertEqual({error, module_not_found}, foil:lookup(erl_optics, optics)).
+
+counter_inc_test() ->
+    start(),
+    erl_optics:counter_inc(<<"counter">>),
+    foil_reload(),
```

Comment:
Would it make sense to have an assert here for `<<"counter">>` not being in foil, as it would be in the ETS table instead?

Reply:
I agree, but I think it warrants its own test, which I added under the foil_reload_test name.